schema.json

```diff
@@ -2,7 +2,7 @@
     "openapi": "3.0.2",
     "info": {
         "title": "OpenAPI Schema for Salesforce Functions invocation contexts",
-        "version": "1.0.0",
+        "version": "1.0.1",
```

Comment:
need version change in pom.xml in java if you are planning to use the latest version in core,
also need a new tag created for publishing to maven-central (public)

https://github.com/forcedotcom/sf-fx-schema#readme

Reply:
I will do the publishing and then change core.

Reply:
For the `pom.xml` version updates per the README, that change would be done on the release branch after merging this change to `main`.  So I think this PR is OK.